src/inspect.rs

```diff
@@ -1,18 +1,161 @@
 use anyhow::{anyhow, Result};
+use kubewarden_policy_sdk::metadata::ProtocolVersion;
+use mdcat::{ResourceAccess, TerminalCapabilities, TerminalSize};
 use policy_evaluator::policy_metadata::Metadata;
+use prettytable::{format::FormatBuilder, Table};
+use pulldown_cmark::{Options, Parser};
+use syntect::parsing::SyntaxSet;
 
-pub(crate) fn inspect(uri: &str) -> Result<()> {
+pub(crate) fn inspect(uri: &str, output: Option<&str>) -> Result<()> {
```

Comment:
I would be in favour of having `output: OutputType`, being `OutputType` an enum that has a `TryFrom<&str>` and enumeration values of `Pretty` and `Yaml` for example. Thus, on the CLI side, we can construct a OutputType::try_from(output_type)`, being `output_type` the argument from the user. This way, this call can focus on the enum values we allow instead of having to match on `Option<&str>`.

Reply:
Good idea! I'll work on that

Reply:
This `output` can now take an `OutputType`, right? :)